src/integrations/prefect-docker/prefect_docker/deployments/steps.py

```diff
@@ -127,6 +127,8 @@ def build_docker_image(
     tag: str | None = None,
     additional_tags: list[str] | None = None,
     ignore_cache: bool = False,
+    persist_dockerfile: bool = False,
+    dockerfile_output_path: str | None = None,
```

Comment:
nit: this will simplify the code a little bit.
```suggestion
    dockerfile_output_path: str = "Dockerfile.generated",
```